cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -146,9 +146,9 @@ endif
 # Depending on the version of modelsim the interfaces that it supports can change
 # Append or remove values to INT_LIBS depenending on your license
 results.xml: $(SIM_BUILD)/runsim.do $(COCOTB_LIBS) $(INT_LIBS)
-	set -o pipefail; cd $(SIM_BUILD) && $(LIB_LOAD) MODULE=$(MODULE) TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) COCOTB_SIM=1 \
+	set -o pipefail; $(LIB_LOAD) MODULE=$(MODULE) TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) COCOTB_SIM=1 \
 	GPI_EXTRA=$(GPI_EXTRA) TOPLEVEL_LANG=$(TOPLEVEL_LANG) PYTHONPATH=$(LIB_DIR):$(PWD):$(NEW_PYTHONPATH) \
-	$(CMD) $(PLUSARGS) -do runsim.do 2>&1 | tee sim.log
+	$(CMD) $(PLUSARGS) -do ${SIM_BUILD}/runsim.do 2>&1 | tee sim.log
```

Comment:
What's the issue with starting questa from PWD and having all libraries and temporary files there as well, i.e. get rid of sim_build entirely. That's what we do for other simulators, so it's the expected behavior (it's not beautiful, but at least consistent).

Reply:
Well, the current behavior is that questa is started from sim_build and loads a script that lives there (runsim.do).

This commit changes things so the generated script is still in sim_build but questa starts from PWD. I thought that was more consistent with the other simulators-- Cadence, at least, puts a log file and the compiled libraries (the "INCA_libs" directory) in sim_build.

But we could put everything in PWD if that makes more sense.

Reply:
BTW: one can do without do file but with -do argument like here: https://github.com/themperek/cocotb-test/blob/54602a6a954a46d9d6583e897a7d814b8d408022/cocotb_test/simulator.py#L298 but not sure if this can be done with Makefile.